P/PETSc/bundled/patches/mpi-constants.patch

```diff
@@ -0,0 +1,239 @@
+diff --git a/src/mat/impls/aij/mpi/superlu_dist/superlu_dist.c b/src/mat/impls/aij/mpi/superlu_dist/superlu_dist.c
```

Comment:
Would it be possible to upstream this patch?  It's a largish, upgrading petsc would be complicated if we shipped invasive patches.

Reply:
I will do that.

Is it okay to keep the patch for the time being?

Reply:
Yes, but going forward we always prefer needing as few patches as possible